src/hotspot/share/opto/vectorization.cpp

```diff
@@ -46,7 +46,7 @@ VPointer::VPointer(const MemNode* mem,
 #endif
   _nstack(nstack), _analyze_only(analyze_only), _stack_idx(0)
 #ifndef PRODUCT
-  , _tracer((phase->C->directive()->VectorizeDebugOption & 2) > 0)
+  , _tracer(phase->C->directive()->traceautovectorization_tags().at(TraceAutoVectorizationTag::POINTER_ANALYSIS))
```

Comment:
I now do the ugly thing. Later, with the bigger refactoring, I will pass `VLoop` into the `VPointer`, and then we can access the flag via `VPointer -> VLoop -> VTrace`.